packages/lint/package.json

```diff
@@ -6,8 +6,8 @@
   "module": "dist",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "dx-tools build-lib src dist --failOnError",
-    "clean": "dx-tools clean dist",
+    "build": "dx-tools build-lib src dist --failOnError | exit 0",
```

Comment:
How does this change affect build?

Reply:
After removing packages/lint/src/disallowImportsRule.ts in lint-package his "src"-folder became empty.
After removing, `yarn build`,  `yarn clean` and `yarn prepare` in linter-package return errors.
Therefore, the execution of the script `yarn lerna run prepare` in the main-package also return an error.
We can remove `build`,  `clean` and `prepare` in linter-package, but we anyway rollback this changes in #162

Now I override calling `build` and `clean` as success with forced return 0.
I make it for save build/prepare processes "as is".
In #162 we will rollback this override.

We will remove or save `build`,  `clean` and `prepare` scripts in linter-package?

Reply:
I would keep the package as clean as possible. If we need any removed code/configuration further we can always pick it from git history.

Reply:
Done.